core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -802,6 +802,17 @@ private CarbonCommonConstants() {
    */
   public static final String DEFAULT_SEGMENT_LEVEL_THRESHOLD = "4,3";
 
+
+  /**
+   * Min value of Number of segments to be merged in levels
```

Comment:
What does `in levels` mean? Can you rephrase it?

Reply:
solved